app/graphql/types/query_type.rb

```diff
@@ -1,12 +1,11 @@
 module Types
   class QueryType < Types::BaseObject
-    # Add root-level fields here.
-    # They will be entry points for queries on your schema.
+    field :get_album, AlbumType, null: false do
```

Comment:
In the card the query is:

```query {
  album (id) {
    id
    title
    artist
    photos
  }
}
```

But you have a query called `get_album` instead of `album`